runtime-js/Integer/nearestFloat.js

```diff
@@ -0,0 +1 @@
+return Float(this.valueOf());
```

Comment:
This doesn't work yet, see the [discussion to a previous commit][https://github.com/ceylon/ceylon.language/pull/761#discussion_r39931649].